### DISCUSSION TITLE
Exclude via cut number DRC check for pin access on a particular std cell

### DISCUSSION THREAD

**Discussion by marcopoles** [The-OpenROAD-Project/OpenROAD#4140] [Q&A]
Hi ,
I'm working with foreign tech that has the std cell BUFFX16 with huge M1 shape for output pin. 
During pin access routing, the DRC rule on min via1 cut is not satisfied, so the pin has no access point stopping the flow.
I read on other discusstion that is not yet possible to place multiple via cut, so, could be possible to exclude the drc check for minimum cut number to use for a list of std cells?

An alternative approach should be modify the LEF using a thinner M1 shape, correct?

Thanks

**Comment by maliberty** [comment]
Have you verified that the problem is min cut and not another rule?  If so, you might just exclude that cell from being used putting it in the don't use list.

**Reply by marcopoles** [reply]
Yes, I verified that modified the LEF with a smaller M1 the routing is done. 
In the approach to add the cell to don't use list, is the BUFFX16 replaced with 2 BUFFX8 in parallel for example?

**Reply by maliberty** [reply]
It would depend on the circumstance but that's possible.  You could redesign the pin but be sure to update the gds as well.
